library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -79,6 +79,20 @@
   }
 }
 
+@media (min-width: 767px) {
+  .sidebar-site-title {
+    width: 128px;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
```

Comment:
I think you should use line-clamp here, according to MDN.

https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

Reply:
I don't understand. Can you please elaborate?

Reply:
From the MDN link: "This property was originally implemented in WebKit and has some issues."
"The [CSS Overflow Module Level 4](https://drafts.csswg.org/css-overflow-4/#propdef-line-clamp) specification also defines a line-clamp property, which is meant to replace this property and avoid its issues."

Reply:
How about we use both? I can't use just line-clamp in WebKit-based browsers i.e. Chrome and Safari

Reply:
I would also use both. It seems the quoted spec is still a draft and the [browser support on "can i use"](https://caniuse.com/css-line-clamp) suggests using the `-webkit-` prefix.